frb/galaxies/frbgalaxy.py

```diff
@@ -274,27 +274,69 @@ def gen_cigale_data_in(self, ID=None, filename='data.fits', overwrite=False):
         # Convert DES and SDSS fluxes to mJy
         ABmagbands = ["DES_"+band for band in defs.DES_bands]
         ABmagbands += ["SDSS_"+band for band in defs.SDSS_bands]
+        ABmagbands += ['VLT_'+band for band in defs.VLT_bands]
+        #ABmagbands += ['VISTA_'+band for band in defs.VISTA_bands]
```

Comment:
Is VISTA implemented or not?  I see it in `defs.py` but it is commented out here.
Either is fine, but it looks like it is a mixed implementation.

Reply:
It has been implemented below. I removed VISTA from the list of AB magnitude bands as their magnitudes are in the Vega system. There is a snippet below in the file that converts VITS mags to fluxes.